backend-tests/tests/test_inventory.py

```diff
@@ -108,48 +114,11 @@ def make_pending_device(utoken, tenant_token=""):
     return dev
 
```

Comment:
Se can remove `make_pending_device` from this file too.

Reply:
fixed